plaso/cli/image_export_tool.py

```diff
@@ -490,33 +495,34 @@ def _ReadSpecificationFile(self, path):
     """
     specification_store = specification.FormatSpecificationStore()
 
-    with open(path, 'rb') as file_object:
+    with io.open(
+        path, 'r', encoding=self._SPECIFICATION_FILE_ENCODING) as file_object:
       for line in file_object.readlines():
         line = line.strip()
-        if not line or line.startswith(b'#'):
+        if not line or line.startswith('#'):
           continue
 
         try:
           identifier, offset, pattern = line.split()
         except ValueError:
-          logger.error('[skipping] invalid line: {0:s}'.format(
-              line.decode('utf-8')))
+          logger.error('[skipping] invalid line: {0:s}'.format(line))
           continue
 
         try:
           offset = int(offset, 10)
         except ValueError:
-          logger.error('[skipping] invalid offset in line: {0:s}'.format(
-              line.decode('utf-8')))
+          logger.error('[skipping] invalid offset in line: {0:s}'.format(line))
           continue
 
         try:
-          pattern = pattern.decode('string_escape')
+          if py2to3.PY_2:
+            pattern = codecs.decode(pattern, 'string_escape')
+          else:
+            pattern = codecs.escape_decode(pattern)[0]
```

Comment:
since `codecs.escape_decode` should work on Python 2 as well, remove the if-else or add a comment to so at a later point in time?